data/tribes/initialization/amazons/units.lua

```diff
@@ -1391,7 +1391,9 @@ wl.Descriptions():new_tribe {
          name = "amazons_stone_workshop",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an Amazon production site: Stone Workshop
-            purpose = pgettext("amazons_building", "Makes tools and spears for our soldiers.")
+            purpose = pgettext("amazons_building", "Makes tools and spears for our soldiers."),
+            -- TRANSLATORS: Performance helptext for an Amazon production site: Stone Workshop
+            performance = pgettext("amazons_building", "If this building is fully supplied and all tools and spears are needed by the economy, production of one of each tool takes %1$s and one of each spear takes %2$s on average. If only one kind of tool or spear is needed by the economy, its production takes %3$s on average."):bformat(format_minutes_seconds(21, 18), format_minutes_seconds(7, 6), format_minutes_seconds(1, 11))
```

Comment:
if all wares are needed the time is different as in the time of the full cycle more spears are produced. so a full cycle has 3 of each spear but only 1 of each tool

Reply:
So it's similar to the "smokery" issue, only more complicated. Anyhow, needs better explanation.

Reply:
Ok, so how about "producing one of each tool and 3 of each spear takes…, yielding one of each spear in … on average"?

Reply:
Well I believe we agreed on the wording scheme before.    
Furthermore I agree with @gpalino that in the end the important information of this section is the average time to produce a ware. I think it is simply not really possible to explain every detail of our production in detail.     
the pure average time described here and calculated as precise as possible would accomodate for the purpose given by @gpalino in the first post.  
last argument: if we start to explain this in detail here we would probably need to explain all cases of the atlanteans crystal mine as well which would be just an overkill imho.

Reply:
The agreed wording works well when full cycle means one of each product. (if we can use a common word for them :) I didn't really notice the problem with the crystal mine either, probably because there the products are all very different, and all have different amounts in the full cycle. (now I see that by-products are not mentioned in the single need times, but that's indeed a good thing)

So maybe it's OK, but https://github.com/widelands/widelands/pull/5861#discussion_r1199819149 is also based on the agreed wording, and extends it to this case and could be adopted for smokeries, smelters, and empire armoursmith too. @gpalino What do you think?

Reply:
In this case I prefer shorter text without extension - all important information is already in the text and average numbers.